keyboards/handwired/not_so_minidox/pro_micro.h

```diff
@@ -0,0 +1,362 @@
+/*
```

Comment:
Why do this? 

For the most part, you should be able to use the pins, without having to refine things here.

Reply:
This was left over from a fork of someone else's fork. removed.

Reply:
Ah, okay.